controllers/front/showsavedcreditcards.php

```diff
@@ -37,7 +37,7 @@ public function initContent()
                     $savedCreditCard[] = [
                         'userID' => $result['userID'],
                         'creditCard' => $result['creditCardNumber'],
-                        'cardName' => $result['cardName'],
+                        'cardName' => $result['cardBrand'],
                         'cardBrand' => $result['cardBrand'],
```

Comment:
Is this expected to change cardName key to use cardBrand value?

Reply:
I have removed this entry as we no longer use this variable in the template file.